spec/stud/interval_spec.rb

```diff
@@ -1,8 +1,19 @@
 require "stud/interval"
 require "spec_env" # from the top level spec/ directory
 
+module Stud
+  def reset!(target = Thread.current)
```

Comment:
yes, I know. I needed this for the specs and did not want to introduce a "public" `reset!`api  just for specs ... so I added it there.